tox.ini

```diff
@@ -1,10 +1,11 @@
 [tox]
-minversion = 2.0
-envlist = py35
+minversion = 3.2.0
+envlist = py36,py37,py38,pep8
+ignore_basepython_conflict = true
 
 [testenv]
+basepython = python3
```

Comment:
I think we can drop this, can't we?

Reply:
I wondering that will happen if some people use tox on environment that still have python 2 installed and `basepython` is not specified...

I'm not sure about this use case so I prefered keep this one just in case.

Thoughts?

Reply:
Indeed on the CI point of view  we could drop it but for local testing I think we should keep it.

Reply:
The proper test environments all use version-specific names, right? So they would automatically select the python version (that's why we need the flag to report an error with overriding basepython if that value does not match the requested test environment).

Reply:
I got it, you're right, I'll drop this.